catroid/src/main/res/layout/brick_loop_endless.xml

```diff
@@ -43,6 +43,6 @@
             android:id="@+id/brick_loop_endless_label"
             style="@style/BrickText.SingleLine"
             android:text="@string/brick_loop_end" />
-    </LinearLayout>
+    </org.catrobat.catroid.ui.BrickLayout>
 
 </LinearLayout>
```

Comment:
as in Robert C. Martin's ~Clean Code~: always leave a file in a better shape than you found it, right? :smiley: